internal/layers/lcow.go

```diff
@@ -0,0 +1,285 @@
+//go:build windows
```

Comment:
I assume this file is just rearranged code? Anything interesting to review here?

Reply:
The first commit only splits the LCOW & WCOW code into two files so no need to review it. However, the 3rd commit ("refactor layer mount functions" : 318b67e43f6c977a5e1a1d98edd0859d842ff039) moves the layer parsing function for LCOW from `containerd-shim-runhcs-v1` package to the `layers` package. However, it doesn't modify that code so yeah, it's just rearranging the code.